### DISCUSSION TITLE
rework speed and alt tape fonts

### DISCUSSION THREAD

**Discussion by giraffemecrazy** [magknight/787aviatorsTracker#826] [Ideas]
Hi, i was looking at the PFD and noticed that the fonts are off, i was wondering if this could be looked at for in a future update, something that the tapes also miss are the numbers sliding when you reach a new thousand number, it wont just jump but it will 'slide' into place. Some videos have great examples of this.

**Comment by aeroniemi** [comment]
we won't be changing the typefaces - there's a whole series of reasons that isn't possible
we could concider doing the numeric sliding for 1000 though

**Comment by giraffemecrazy** [comment]
i'd be awesome if the animation for the numbers could be done. I just looked at the file structure of the 78, and i was curious if you could list a reason as to why the font wont be changed?

**Reply by Simleybob** [reply]
Licensing etc

**Reply by giraffemecrazy** [reply]
thats rather interesting as other unlicensed aircraft do have the correct fonts.
